### PR TITLE
Add chain link msg docs

### LINKED ISSUE #589: Missing chain-link docs
Looking at the docs I've found out that there's no docs for the chain-links related transactions.
I think it should be better to add it once the generate-json command is completed.

### PR DESCRIPTION
**Pull request by dadamu**
## Description

Add docs associated chain link.
Close #589

## Checklist
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [ ] Wrote integration tests (simulation & CLI). 
- [ ] Updated the documentation. 
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: docs/developers/msgs/profiles/link-chain-account.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    },
+    "chain_config": {
+        "name": "<Name of the target external chain>"
+    },
+    "proof": {
+        "pub_key": {
+            "@type": "/cosmos.crypto.secp256k1.PubKey",
+            "value": "<Base64 encoded public key>"
+        },
+        "signature": "<Hex encoded signature created with private key associated given public key>",
```
Comment (bragaz) [inline docs/developers/msgs/profiles/link-chain-account.md:23]:
```suggestion
        "signature": "<Hex encoded signature created with private key associated to the given public key>",
```

File: docs/developers/msgs/profiles/link-chain-account.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        "plain_text": "<Text for getting signature by signing with private key>"
```
Comment (bragaz) [inline docs/developers/msgs/profiles/link-chain-account.md:24]:
```suggestion
        "plain_text": "<Text signed with private key>"
```

File: docs/developers/msgs/profiles/link-chain-account.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    },
+    "signer": "<Desmos address of the profile witch link the chain account>"
```
Comment (bragaz) [inline docs/developers/msgs/profiles/link-chain-account.md:26]:
```suggestion
    "signer": "<Desmos address of the profile linking the chain account>"
```

File: docs/developers/msgs/profiles/link-chain-account.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+### Attributes
+
+| Attribute | Type | Description |
+| :-------: | :----: | :-------- |
+| `chain_address` | [AddressData](../../types/profiles/chain-link.md#Address) | Address data of the external chain account |
+| `chain_config` | [ChainConfig](../../types/profiles/chain-link.md#ChainConfig) | Details of the target external chain |
+| `proof` | [Proof](../../types/profiles/chain-link.md#Proof) | Data proving the ownership of the external chain account |
+| `signer` | String | Desmos address of the profile to which the link will be associated |
```
Comment (bragaz) [inline docs/developers/msgs/profiles/link-chain-account.md:37]:
```suggestion
| `signer` | String | Desmos address of the profile with which the link will be associated |
```

**Review by bragaz — COMMENTED** [review]
Small fixes but looks ok overall

**Review by bragaz — APPROVED** [review]
👍

**Comment by bragaz** [comment]
@RiccardoM Can we merge this? It will be useful for the mainnet

**Review by RiccardoM — APPROVED** [review]

**Comment by RiccardoM** [comment]
> @RiccardoM Can we merge this? It will be useful for the mainnet

Sure. Thanks for the ping and sorry for the delay 🙏

**Pull request merged**